docs/reference/modules/cluster/disk_allocator.asciidoc

```diff
@@ -75,13 +75,25 @@ Defaults to `true`. Set to `false` to disable the disk allocation decider.
 Controls the low watermark for disk usage. It defaults to `85%`, meaning that {es} will not allocate shards to nodes that have more than 85% disk used. It can alternatively be set to a ratio value, e.g., `0.85`. It can also be set to an absolute byte value (like `500mb`) to prevent {es} from allocating shards if less than the specified amount of space is available. This setting has no effect on the primary shards of newly-created indices but will prevent their replicas from being allocated.
 // end::cluster-routing-watermark-low-tag[]
 
+`cluster.routing.allocation.disk.watermark.low.max_headroom` {ess-icon}::
```

Comment:
So that I do not forget, I am adding a comment here as well for this `ess-icon`. Do we need to update the ESS team for the new settings before or once this PR is merged? If yes, how should I/we do it? Thanks for any tips @DaveCTurner , @henningandersen .

Reply:
We'd need to add these settings to the list of settings that users are permitted to adjust if that's what we want, but I think we don't want that: users should be using the default watermarks in ESS.

Reply:
Oh I see. I do see the icon set for the frozen max headroom in master, see a few lines below:

```
`cluster.routing.allocation.disk.watermark.flood_stage.frozen.max_headroom` {ess-icon}::
```

Should we remove it from there as well? Or would it be a breaking change -- and in this case, should we follow the same approach? cc @henningandersen

Reply:
That's strange. I don't think users are allowed to adjust `cluster.routing.allocation.disk.watermark.flood_stage.frozen.max_headroom` in ESS. Maybe the icon means something else...

Reply:
I think your interpretation of the icon is correct, see #57953. I think I added it in error when adding the frozen flood stage settings. It is not allowed in ESS and I also think it should not be (likewise for the new settings).

I think removing the icon for `cluster.routing.allocation.disk.watermark.flood_stage.frozen` and `cluster.routing.allocation.disk.watermark.flood_stage.frozen.max_headroom` in this PR is fine (also fine to defer to a separate PR if you prefer).

Reply:
Now questioning myself on this. Since the frozen flood stage settings are dynamic they could be set by users today. Once we move them all to be operator-dynamic instead, we can remove the icon. The settings are not settable in yaml in ESS though, but since they can be set through APIs, I think the icon should be included.

CC: @lockewritesdocs to ensure we understand the meaning of the icon correctly.

Reply:
@henningandersen, the icon means that a setting is supported in ESS, based on the ESS allowlist. If you hover the icon, there's a tooltip, though I wonder how many users stop to actually hover that icon.

<img width="760" alt="image" src="https://user-images.githubusercontent.com/25848033/181821427-ebbc8742-4830-428d-a9c9-e6a914606f46.png">

Reply:
Hi @henningandersen , as we discussed, I removed the ess-icon from the new settings, but left it intact (as it was) for the frozen max headroom setting and you will follow up in a next PR for handling that correctly. Please feel free to make an issue not to forget. I will close this conversation as resolved, but feel free to unresolve it and provide more feedback if needed. Thanks!